packages/kit/test/apps/basics/src/routes/load/url-hash/+page.js

```diff
@@ -1,3 +1,6 @@
+// TODO: remove this when Svelte 5 hydration is fixed. See https://github.com/sveltejs/svelte/issues/14437
+export const ssr = false;
```

Comment:
temporarily disabled hydration for this test so that the error page can render correctly